app/controllers/cart_controller.rb

```diff
@@ -22,12 +22,30 @@ def remove_item
     redirect_to '/cart'
   end
 
+  def edit_quantity
+    adjust_quantity
+  end
+
   private
 
   def require_not_admin
       render file: "/public/404" if current_admin
   end
 
+  def item
+    @item = Item.find(params[:item_id])
+  end
+
+  def adjust_quantity
+    if params[:quantity] == "add" && item.inventory > session[:cart][params[:item_id]]
+      session[:cart][params[:item_id]] += 1
+      redirect_to '/cart'
```

Comment:
This was all based off the code left behind at the bottom that was commented out